docs/okeps/okep-5259-no-overlay.md

```diff
@@ -0,0 +1,1491 @@
+          # the external BGP route reflector
+          - address: 172.20.0.2
+            asn: 64512
+            disableMP: true
+            toReceive:
+              allowed:
+                mode: filtered
+                prefixes:
+                # accept pod subnets of the default network explicitly
+                - prefix: 10.128.0.0/16
+                  ge: 24
+    ```
+
+3. The cluster admin shall create a RouteAdvertisements CR to advertise the
+   default network. Make sure the RouteAdvertisements CR selects the
+   FRRConfiguration defined in the previous step.
+
+    ```yaml
+    apiVersion: k8s.ovn.org/v1
+    kind: RouteAdvertisements
+    metadata:
+      name: default
+    spec:
+      advertisements:
+      - PodNetwork
+      # Select the FRRConfiguration defined in step-2 with the custom label.
+      frrConfigurationSelector:
+        matchLabels:
+          network: default
+      networkSelectors:
+      - networkSelectionType: DefaultNetwork
+      # The empty nodeSelector selects all nodes. We don't support a network in a overlay and no-overlay hybrid mode.
+      nodeSelector: {}
+    ```
+
+4. The cluster admin enables no-overlay mode by running ovn-kubernetes with
+   config file below:
+
+    ```ini
+    [default-network]
+    transport = no-overlay
+    no-overlay-routing = unmanaged
+    no-overlay-outbound-snat = enable
+    ```
+
+5. The default network controller in ovnkube-controller will create the OVN
+   virtual topology for the default network. Neither the transit switch nor the
+   static routes to remote pod subnets will be created.
+
+6. The default network controller in ovnkube-cluster-manager shall watch the
+   RouteAdvertisements CRs. It shall ensure at least one RouteAdvertisements CR
+   is correctly configured to advertise the pod networks and will validate the
+   RouteAdvertisements CR status. If the RouteAdvertisements CR is missing or
+   its status is not accepted, it shall report event and log an error message.
+
+7. The BGP feature is responsible for exchanging BGP routes to remote pod
+   subnets between nodes. Ultimately, these routes will be imported into the
+   gateway router.
+
+#### Enable No-overlay Mode for the Default Network with Managed Routing
+
+1. The frr-k8s pods shall be deployed on each node.
+
+2. In the Managed routing mode, the default network controller in
+   ovnkube-cluster-manager will create the RouteAdvertisements CR and the base
+   FRRConfiguration CR automatically. The cluster admin only needs to start
+   ovn-kubernetes with the following flags:
+
+    ```ini
+    [default-network]
+    transport = no-overlay
+    no-overlay-routing = managed
+    no-overlay-outbound-snat = disable
+
+    [internal-bgp-fabric-options]
+    topology = full-mesh
+    as-number = 64514
+    ```
+
+3. The default network controller in ovnkube-cluster-manager will create a base
+   FRRConfiguration CR named `default` in the `frr-k8s-system` namespace, and a
```

Comment:
No, the FRRConfiguration is created by whatever controller is in charge of creating the internal BGP fabric. ItWhy would it be the default network controller in ovnkube-cluster-manager?
Also I would use more specific names than just `default` as they can collide with other instances. For example the RA controller is using `ovnk-generated-blabla` to avoid collisions